modin/core/execution/ray/common/utils.py

```diff
@@ -50,6 +50,8 @@
 _RAY_IGNORE_UNHANDLED_ERRORS_VAR = "RAY_IGNORE_UNHANDLED_ERRORS"
 
 ObjectIDType = ray.ObjectRef
+# TODO: Minimum version of Ray - 1.13
+# `if` branch can be deleted
```

Comment:
Good catch. We can also remove IsExperimental import in this file.
https://github.com/modin-project/modin/blob/324099d8737ead092e4acacfe03b1caa82e01986/modin/utils.py#L52C34-L52C48

Reply:
I wonder why `flake8` does not see that this import is not used. Hm

Reply:
@YarShev https://github.com/modin-project/modin/pull/6826

Reply:
thanks